api-report/test-runtime-utils.api.md

```diff
@@ -69,7 +74,7 @@ export interface IMockContainerRuntimePendingMessage {
 export class InsecureTokenProvider implements ITokenProvider {
     constructor(
     tenantKey: string,
-    user: IUser,
+    user: IInsecureUser,
```

Comment:
Is this considered a breaking change since we now require the `name` field?

Reply:
This is essentially a breaking change. We chatted about this topic in our office hours yesterday. Whether the package is a test package or not (FFX for example consumes some of our test packages), we should treat them the same. So more appropriate place for this kind of change would be next branch.

Reply:
done, base now changed to `next`

Reply:
I think we also need to add a note to `BREAKING.md` about this, with necessary migration instructions.

Reply:
How about now?

Reply:
Looks great! Thanks!